text/0000-build-std/5-stage-1b.md

```diff
@@ -0,0 +1,986 @@
+# Proposal
+[proposal]: #proposal
+
+Users can now optionally declare explicit dependencies on the standard library
+in their `Cargo.toml` files ([?][rationale-why-explicit-deps]):
+
+```toml
+[package]
+name = "hello_world"
+version = "0.1.0"
+edition = "2024"
+
+[dependencies]
+std = { builtin = true }
+```
+
+`builtin` is a new source of dependency, like registry dependencies (with the
+`version` key and optionally the `registry` key), `path` dependencies or `git`
+dependencies. `builtin` can only be set to `true` and cannot be combined with
+any other dependency source for a given dependency
+([?][rationale-builtin-other-sources]). `builtin` can only be used with crates
+named `core`, `alloc` or `std` ([?][rationale-no-builtin-other-crates]) on
+stable, but can be with any crate name on nightly
+([?][rationale-nightly-builtin-crates]).
```

Comment:
not just nightly but with a `cargo-feature`, right?

Reply:
Yes

Reply:
Thanks